### ISSUE TITLE
Defining own testcase

### ISSUE THREAD

**Issue by turtlebot1** [AlvinYH/Faster-VoxelPose#40]
Hello authors,
Thank you for sharing the code of this amazing work. I am using your framework to solve a multi-person multi-camera problem in real-time environment. I wanted to ask if we need to train the model again with our own data after setting up the environment or if we could use the pre-trained model and test it on the new data.
It would be really helpful if you could help me out in this aspect. 

Thank you for your time.

**Comment by 342600694** [comment]
> Hello authors, Thank you for sharing the code of this amazing work. I am using your framework to solve a multi-person multi-camera problem in real-time environment. I wanted to ask if we need to train the model again with our own data after setting up the environment or if we could use the pre-trained model and test it on the new data. It would be really helpful if you could help me out in this aspect.
> 
> Thank you for your time.

Hello, have you solved the problem of human body pose estimation for multiple people and multiple cameras in a real-time environment?

**Comment by turtlebot1** [comment]
@342600694 ,
Yes, I have solved the problem of setting up this model for a custom environment and get the results. I have not made it work real-time and am currently finding a way to decrease the latency but I am getting ok-ish results by testing the inference on a custom setup.

**Comment by 342600694** [comment]
> @342600694 , Yes, I have solved the problem of setting up this model for a custom environment and get the results. I have not made it work real-time and am currently finding a way to decrease the latency but I am getting ok-ish results by testing the inference on a custom setup.

I am currently reading the author's code and may try to use my own dataset to test and make it work in real time in the future. If there are any issues in the future, can I ask you for advice under this post? Thank you very much.

**Comment by turtlebot1** [comment]
@342600694 
Sure, just be careful with the camera calibration process and your results would be fine.

**Comment by 342600694** [comment]
> @342600694 Sure, just be careful with the camera calibration process and your results would be fine.

hey bro I can't understand some of the contents in project_individual.py. Is there any reference information?
